src/components/EditItemsList/EditItemsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useEffect' is defined but never used.